src/renderer/components/ProjectCreation/CloudConfigView.tsx

```diff
@@ -15,11 +15,10 @@ import { PrimaryButton, SecondaryButton } from '../Blocks/Buttons';
 import ipc from '../../ipc';
 
 const { openExternalLink, storeCloudCredentials } = ipc;
-
 interface Props {
-  prevView: () => void;
+  prevView: (() => void) | null;
```

Comment:
Does this break the other callers of `prevView` / `nextView`

Reply:
nah I tested it and works fine. There is a check for null before these are called

Reply:
Ok great then